prtpy/partitioning/integer_programming_avg.py

```diff
@@ -0,0 +1,151 @@
+"""
+Produce an optimal partition by solving an integer linear program (ILP).
+
+Programmer: Erel Segal-Halevi, Eitan Lichtman
+Since: 2022-02
+
+Credit: Rob Pratt, https://or.stackexchange.com/a/6115/2576
+"""
+
+from typing import List, Callable, Any
+from numbers import Number
+
+from prtpy import objectives as obj, outputtypes as out, Binner, printbins
+from math import inf
+import mip
+
+def optimal(
+    binner: Binner, numbins: int, items: List[any], relative_values: List[any] = None,
+    copies=1,
+    time_limit=inf,
+    verbose=0,
+    solver_name=mip.CBC, # passed to MIP. See https://docs.python-mip.com/en/latest/quickstart.html#creating-models.
+    # solver_name = mip.GRB, # passed to MIP. See https://docs.python-mip.com/en/latest/quickstart.html#creating-models.
+    model_filename=None,
+    solution_filename=None,
+):
+    """
+     Produce a partition that minimizes the sum of the positive differences from the avg, by solving an integer linear program (ILP).
+
```

Comment:
The description of the params in the function documentation does not match the actual params of the function

Reply:
Fixed.